public/app/plugins/datasource/cloudwatch/components/ConfigEditor.tsx

```diff
@@ -121,16 +121,18 @@ export class ConfigEditor extends PureComponent<Props, State> {
         <div className="gf-form-group">
           <div className="gf-form-inline">
             <div className="gf-form">
-              <InlineFormLabel className="width-14">Auth Provider</InlineFormLabel>
+              <InlineFormLabel
+                className="width-14"
+                tooltip="Which AWS credentials chain to use. AWS SDK Default is the recommended option for EKS, ECS or if you've attached an IAM role to your EC2 instance."
+              >
+                Authentication Provider
+              </InlineFormLabel>
               <Select
                 className="width-30"
                 value={authProviderOptions.find(authProvider => authProvider.value === options.jsonData.authType)}
                 options={authProviderOptions}
                 defaultValue={options.jsonData.authType}
                 onChange={option => {
-                  if (options.jsonData.authType === 'arn' && option.value !== 'arn') {
-                    delete this.props.options.jsonData.assumeRoleArn;
-                  }
```

Comment:
Not sure what the impact is when we remove this if statement. Is this because ARN is no longer part of `authProviderOptions`?. Does the test cases reflect this change?

Reply:
Yes it does! I have removed the conditional part of rendering the `arn` div and updated the snapshots.

The idea is that `arn` isn't actually an authentication provider but rather something you do with your selected authentication provider.